src/Transfer.js

```diff
@@ -0,0 +1,390 @@
+                            onChange,
+                            selectedPlainOptions: selected,
+                            setHighlightedPickedOptions,
+                        })
+                    }
+                />
+            </Actions>
+
+            <RightSide dataTest={dataTest} width={selectedWidth}>
+                <PickedOptions
+                    dataTest={dataTest}
+                    emptySelectionPlaceholder={emptySelectionPlaceholder}
+                    highlightedPickedOptions={highlightedPickedOptions}
+                    toggleHighlightedPickedOption={
+                        toggleHighlightedPickedOption
+                    }
+                    deselectSingleOption={deselectSingleOption}
+                >
+                    {pickedOptions}
+                </PickedOptions>
+
+                {(rightFooter || enableOrderChange) && (
+                    <RightFooter dataTest={dataTest}>
+                        {enableOrderChange && (
+                            <ReorderingActions
+                                dataTest={dataTest}
+                                disabledDown={isReorderDownDisabled({
+                                    highlightedPickedPlainOptions: highlightedPickedOptions,
+                                    selectedPlainOptions: selected,
+                                })}
+                                disabledUp={isReorderUpDisabled({
+                                    highlightedPickedPlainOptions: highlightedPickedOptions,
+                                    selectedPlainOptions: selected,
+                                })}
+                                onChangeUp={() =>
+                                    moveHighlightedPickedOptionUp({
+                                        selectedPlainOptions: selected,
+                                        highlightedPickedPlainOptions: highlightedPickedOptions,
+                                        onChange,
+                                    })
+                                }
+                                onChangeDown={() => {
+                                    moveHighlightedPickedOptionDown({
+                                        selectedPlainOptions: selected,
+                                        highlightedPickedPlainOptions: highlightedPickedOptions,
+                                        onChange,
+                                    })
+                                }}
+                            />
+                        )}
+
+                        {rightFooter}
+                    </RightFooter>
+                )}
+            </RightSide>
+        </Container>
+    )
+}
+
+Transfer.defaultProps = {
+    dataTest: 'dhis2-uicore-transfer',
+    initialFilter: '',
+    selected: [],
+    height: '240px',
+    optionsWidth: '320px',
+    selectedWidth: '320px',
+    maxSelections: 1,
+    filterOptions: defaultFilterOptions,
```

Comment:
This name confused me a bit. I thought it was for additional options you could set for the filter, but I see it's the filtering function. In that case I would have expected maybe optionFilter and defaultOptionFilter. But maybe that's just my expectations.

Reply:
The reason I chose this name is because the general function is called `filter` and this one is a filter function for options. But I get that the word "Options" is ambiguous here, which def. will cause confusion. Maybe `applyFilterToOptions`?

Reply:
Or, I'm not sure if this applies, but filterFn and filterOptionsFn?

Reply:
Now it's `searchTerm` and `filterOptions`.
If we want to make clear that `filterOptions` needs to be a function, I'd prefer `filterCallback`, what do you think?

Reply:
Yeah filterCallback sounds good to me as well 👍

Reply:
Done